www.foia.gov/_layouts/default.html

```diff
@@ -0,0 +1,23 @@
+---
+---
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
```

Comment:
I wonder if we should either call this something like `legacy-layout.html` template, or templatize this further to allow for easy extraction of the existing code. Seems like once we start porting over new design and structure, having this in the default template might be confusing.